doc/storage_data_plane_commands_auth.md

```diff
@@ -0,0 +1,74 @@
+# Storage Data Plane Commands Auth
+
+Azure operations can be divided into two categories - control plane and data plane. By running storage data plane commands, requests will be sent directly to Storage service instead of being sent to ARM(Azure Resource Manager). Hence additional credentials are required for successfully auth to Storage service.
+
+## Authorization Options
+
+There are several ways to authorize for storage data plane commands:
+
+### Azure Active Directory integration
+
+By specifying `--auth-mode login`, CLI will use the OAuth token returned by AAD(Azure Active Directory) to authorize requests against Storage service.
+
+Here's an example of getting properties of a blob in a storage account with AAD token
+```cmd
+# Get blob properties using AAD token
+> az storage blob show --name $blobName --container-name $containerName --account-name $accountName --auth-mode login
+```
+
+### Shared key authorization
+
+By specifying `--account-key $accountKey` or `--connection-string $connectionString` configured using account key, CLI can use Shared Key by signing each request with the storage account access key.
+
+Here's an example of getting properties of a blob in a storage account with account key
+```cmd
+# Get storage account access key
+> $accountKey = az storage account keys list --account-name $accountName --resource-group $rgName --query "[0].value"
```

Comment:
By default, we provide multiple-command script in Bash (which is the default of Cloud Shell). If this is for PowerShell, we'd better explicitly call that out.

Reply:
Changed to Bash now